src/app/faq-report-issue/faq-report-issue.page.ts

```diff
@@ -439,17 +448,13 @@ export class FaqReportIssuePage implements OnInit, OnDestroy {
       correlationlist.push({ id: this.formValues.subcategory, type: CorReleationDataType.SUBCATEGORY }) : undefined;
     if (this.formValues && this.formValues.children && this.formValues.children.subcategory) {
       // Board
-      this.formValues.children.subcategory.board && this.formValues.children.subcategory.board.name ?
-        correlationlist.push({ id: this.formValues.children.subcategory.board.name, type: CorReleationDataType.BOARD }) : undefined;
+      correlationlist.push({ id: this.extractPrepareedFieldStr('board'), type: CorReleationDataType.BOARD });
       // Medium
-      this.formValues.children.subcategory.medium && this.formValues.children.subcategory.medium.name ?
-        correlationlist.push({ id: this.formValues.children.subcategory.medium.name, type: CorReleationDataType.MEDIUM }) : undefined;
+      correlationlist.push({ id: this.extractPrepareedFieldStr('medium'), type: CorReleationDataType.MEDIUM });
       // Grade
-      this.formValues.children.subcategory.grade && this.formValues.children.subcategory.grade.name ?
-        correlationlist.push({ id: this.formValues.children.subcategory.grade.name, type: CorReleationDataType.CLASS }) : undefined;
+      correlationlist.push({ id: this.extractPrepareedFieldStr('grade'), type: CorReleationDataType.CLASS });
       // Subject
-      this.formValues.children.subcategory.subject && this.formValues.children.subcategory.subject.name ?
-        correlationlist.push({ id: this.formValues.children.subcategory.subject.name, type: CorReleationDataType.SUBJECT }) : undefined;
+      correlationlist.push({ id: this.extractPrepareedFieldStr('subject'), type: CorReleationDataType.SUBJECT });
```

Comment:
What will be the cdata.id if  user is selected more than one subject ?

Reply:
Same "Subject"

Reply:
it will be an Array or String ?